### PR TITLE
V5 develop

### PR DESCRIPTION
**Pull request by Ogerets**

### CODE REVIEW

**Comment by SanjoDeundiak** [comment]
@Ogerets 
1. Why would we need test_data.txt, if we have data.json?
2. What's the difference between ImportedKeyEntry and KeyEntry?

**Comment by Ogerets** [comment]
@SanjoDeundiak 

1. We don't need it indeed.
2. ImportedKeyEntry contains imported PrivateKey, not Data, and don't have name, but still has meta. It provides the return type of PrivateKeyStorage's load function. The idea of PrivateKeyStorage wrapper, I thought, was to exclude work with PrivateKeyExporter for facilitate reason.

**Comment by SanjoDeundiak** [comment]
@Ogerets PrivateKeyEntry would sound much better instead, as ImprotedKeyEntry doesn't really show what the intend is

**Comment by SanjoDeundiak** [comment]
@Ogerets renaming file does not rename class...

File: Source/KeyStorage/PrivateKeyStorage/PrivateKeyEntry.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//
+
+import Foundation
+import VirgilCryptoAPI
+
+@objc(VSSImportedKeyEntry) public class ImportedKeyEntry: NSObject {
+    @objc public let privateKey: PrivateKey
+    @objc public let meta: [String: String]?
+
+    public init(privateKey: PrivateKey, meta: [String: String]? = nil) {
```
Comment (SanjoDeundiak) [inline Source/KeyStorage/PrivateKeyStorage/PrivateKeyEntry.swift]: @Ogerets Why not @objc?

Comment (Ogerets) [reply Source/KeyStorage/PrivateKeyStorage/PrivateKeyEntry.swift]: @SanjoDeundiak Maybe we need to make it internal to not allow clients create it? It adds a possibility for substitution PrivateKeyEntry passing through the functions for some reason. Is it good?

Comment (SanjoDeundiak) [reply Source/KeyStorage/PrivateKeyStorage/PrivateKeyEntry.swift]: @Ogerets no reasons to forbid that

File: Source/KeyStorage/PrivateKeyStorage/PrivateKeyStorage.swift [hunk 2]
```diff
@@ -45,12 +45,12 @@ import VirgilCryptoAPI
     /// - Parameter name: stored entry name
     /// - Returns: `ImportedKeyEntry` with imported Private Key and meta
```
Comment (SanjoDeundiak) [inline Source/KeyStorage/PrivateKeyStorage/PrivateKeyStorage.swift]: Changing class name does not update documentation

**Pull request merged**